mcda/utils/utils_for_plotting.py

```diff
@@ -226,11 +235,16 @@ def save_figure(figure: object, folder_path: str, filename: str):
     :param filename: str
     :return: None
     """
+    custom_output_path = os.environ.get('PROMCDA_OUTPUT_PATH')  # check if an environmental variable is set
+    output_directory_path = custom_output_path if custom_output_path else DEFAULT_OUTPUT_DIRECTORY_PATH
```

Comment:
inform the user what was selected?
what if the path does not exists?

Reply:
A logger in now added in `read_matrix` to inform the user where the input matrix has been read.
Similar loggers already existed for the output directories.
The case the paths do not exist should now be solved with try-error blocks.